timelock-server-distribution/var/conf/timelock.distributed.yml

```diff
@@ -0,0 +1,50 @@
+install:
+  paxos:
+    data-directory: "var/data/paxos"
+    is-new-service: false
+  timestampBoundPersistence:
+
+runtime:
+  cluster-config-not-live-reloaded:
+    cluster:
+      uris:
+        - "localhost:8421"
+        - "timelock1.palantir.pt:8421"
+        - "timelock2.palantir.pt:8421"
+        - "timelock3.palantir.pt:8421"
+      security:
+        keyStorePath: "var/security/keyStore.jks"
+        keyStorePassword: "keystore"
+        trustStorePath: "var/security/trustStore.jks"
+    local-server: "localhost:8421"
```

Comment:
funnily this works -- this way we can avoid templating. otherwise the other solution here is to require templating